Legion/VaultOfTheWardens/Tormentorum.lua

```diff
@@ -46,10 +43,14 @@ function mod:OnBossEnable()
 	self:RegisterUnitEvent("UNIT_HEALTH_FREQUENT", nil, "boss1")
 	self:Log("SPELL_CAST_START", "SeedofCorruption", 196208)
 	self:Log("SPELL_CAST_START", "FrighteningShout", 201488)
+	self:Log("SPELL_AURA_APPLIED", "GroundEffectDamage", 199917) -- Shadow Crash
+	self:Log("SPELL_PERIODIC_DAMAGE", "GroundEffectDamage", 199917)
+	self:Log("SPELL_PERIODIC_MISSED", "GroundEffectDamage", 199917)
+	self:Log("SPELL_AURA_APPLIED_DOSE", "FleshToStoneApplied", 203685)	

```

Comment:
Trailing whitespace in this line (your editor probably has a setting to prevent this).

Reply:
Just call the function ShadowCrash? You're only warning for one thing.